src/oop/programs_with_multiple_classes/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,42 @@
+# oop-programs_with_multiple_classes-medium-q1
+
+**Parsons**
+
+- You want to write a program that models writing tools and stationery objects. Make a Java program with the following requirements:
+
+
+## Pencil
+
+Your `Pencil` class should have the following items:
+
+- Attributes:
+    - An `int` called **hardness**
+    - A `boolean` called **isForSchool**
+
+- Constructors:
+    - A `public Pencil(int, boolean)` constructor to set the
```

Comment:
For this question, instead of just outright giving them what the constructor header is, you should say something more like: 
"A constructor which accepts an integer and a boolean value to initialize the hardness and isForSchool fields and is accessible from other classes"